### PR TITLE
First pass at adding ADR for open source app code.

### PR DESCRIPTION
**Pull request by sarboc**
Add ADR for open source app code, mostly pulled from the [EASi App ADR](https://github.com/CMSgov/easi-app/blob/master/docs/adr/0000-open-source-application-code.md).

### CODE REVIEW

File: practices/appeng/adrs/0003-open-source-application-code.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+and deciders of code contributions,
+open source code allows us
+to accept community input and contributions.
+Sharing code among our internal Truss Engineering practice
+is also how we better our engineering discipline,
+and ultimately better serve our clients.
+
+While sensitive data could potentially be shared through public repositories,
+modern security practices,
+such secret storage separate of code,
```
Comment (jim) [inline practices/appeng/adrs/0003-open-source-application-code.md:48]:
```suggestion
such as secret storage separate of code,
```

File: practices/appeng/adrs/0003-open-source-application-code.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+and cloud services.
+
+## Decision
+
+[Open Sourcing](https://opensource.org/osd)
+allows for the free sharing and modification of code
+and provides a number of benefits to software teams
+and the outside community.
+
+Open Sourcing application code,
```
Comment (jim) [inline practices/appeng/adrs/0003-open-source-application-code.md:30]:
```suggestion
Open Sourcing application code
```

**Review by jim — APPROVED** [review]
This looks good to me.

Do we want to include a reference to what license or specific language to use for projects? I know we have recommendations for our own projects and for gov't ones.

**Comment by pkarman** [comment]
Possibly reference #202 ?

**Pull request merged**